doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -249,7 +249,16 @@ Bluetooth Mesh samples
 Cellular samples
 ----------------
 
-|no_changes_yet_note|
+* LwM2M carrier library: Decreased the fragment size on cellular samples which include the LwM2M carrier library.
+  This is done to ensure FOTA issued from Verizon LwM2M servers has enough room for HTTP headers.
+
+  * :ref:`slm_description`
+
+  * :ref:`http_application_update_sample`
+
+  * :ref:`lwm2m_carrier`
+
+  * :ref:`modem_shell_application`
```

Comment:
```suggestion
* Decreased the fragment size on the following cellular samples that include the :ref:`liblwm2m_carrier_readme` library.

  * :ref:`slm_description`
  * :ref:`http_application_update_sample`
  * :ref:`lwm2m_carrier`
  * :ref:`modem_shell_application`

    This is done to ensure that FOTA issued from Verizon LwM2M servers has enough room for HTTP headers.
```

Reply:
@divipillai, fyi, to be moved later to the release notes.